HPXMLtoOpenStudio/resources/hvac.rb

```diff
@@ -3639,14 +3620,31 @@ def self.calc_sequential_load_fraction(load_fraction, remaining_fraction)
     return sequential_load_frac
   end
 
-  def self.get_sequential_load_schedule(model, value)
-    s = OpenStudio::Model::ScheduleConstant.new(model)
-    s.setName('Sequential Fraction Schedule')
-    if value > 1
-      s.setValue(1.0)
+  def self.get_sequential_load_schedule(model, value, multipliers)
+    values = []
+    multipliers.each do |mult|
+      val = value * mult
+      if val > 1
+        val = 1.0
+      else
+        val = val.round(5)
+      end
+      values.push(val)
+    end
+
+    values << value if values.empty?
+
+    if values.uniq.length == 1
+      s = OpenStudio::Model::ScheduleConstant.new(model)
+      s.setName('Sequential Fraction Schedule')
+      s.setValue(values[0])
     else
-      s.setValue(value.round(5))
+      start_date = OpenStudio::Date.new(OpenStudio::MonthOfYear.new(1), 1, model.getYearDescription.assumedYear)
+      timestep_day = OpenStudio::Time.new(1, 0)
+      ts = OpenStudio::TimeSeries.new(start_date, timestep_day, OpenStudio::createVector(values), '')
+      s = OpenStudio::Model::ScheduleInterval.fromTimeSeries(ts, model).get
```

Comment:
It is crazy to use ScheduleInterval here. The schedule literally just changes value twice during the year, but this creates an EnergyPlus schedule with a value written out for every day of the year, resulting in 1200 lines in the IDF per schedule when it should be 12 lines. Please choose a schedule that collapses better.